packages/compass-schema-validation/src/components/sample-documents/sample-documents.spec.tsx

```diff
@@ -1,23 +1,20 @@
 import React from 'react';
 import { mount } from 'enzyme';
 import { expect } from 'chai';
-import configureStore from '../../stores';
-import SampleDocuments from '../sample-documents';
+import SampleDocuments from '.';
 import { Provider } from 'react-redux';
+import { createBareStore } from '../../stores/store';
 
 describe('SampleDocuments [Component]', function () {
   it('renders a valid and invalid document preview', function () {
+    const store = createBareStore();
```

Comment:
Might not be a pattern we’d really want to adopt, but this was just very easy for converting the existing code